v0/destinations/facebook_pixel/network-handler.js

```diff
@@ -59,6 +59,13 @@ const errorDetailsMap = {
         scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.SCOPE,
         meta: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.META.ABORTABLE
       }
+    },
+    33: {
+      status: 500,
+      statTags: {
+        scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.SCOPE,
+        meta: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.META.RETRYABLE
+      }
```

Comment:
According to the documentation we need to update the access token before retrying the event. So if we send a retryable error from here with 500 responses, it will continue retrying but will not succeed until the access token is updated. Apart from that if we return 400 and abort the event with a proper message like what might be the issue, that will be correct. WDYT?

Reply:
The idea behind sending this `retryable` meta is that, we would want to keep this in-play until someone actually changes the token in configuration so that when the event retries it is successful. But if it makes sense, we can return `400` for this error and then make a decision on how to comprehensively handle this case.

Reply:
With the retriable approach, the event will be retried until or unless the access token gets updated. But events will not be discarded. Now with the second approach events will get discarded and the customer has to update the access token and retry the event as well if possible. With this in mind, the first approach looks safer. So let's stick with that.

Reply:
I think configuration problem should probably be aborted as if not corrected, it could potentially cause bottlenecks in the system. So probably will change this to an abortable kind of case for now. Does this make sense ?

Reply:
Yeah we can do that as well.